api/types/client.go

```diff
@@ -330,6 +330,11 @@ type PluginRemoveOptions struct {
 	Force bool
 }
 
+// PluginEnableOptions holds parameters to enable plugins.
+type PluginEnableOptions struct {
+	Timeout int64
```

Comment:
not sure what we decided for timeouts, but HealthConfig has a `time.Duration` type Timeout, whereas Config has a `*int` type StopTimeout field. So we should try to be consistent with either one.